course_discovery/apps/edly_discovery_app/api/v1/views/dataloader_api.py

```diff
@@ -10,7 +10,7 @@
 from rest_framework.views import APIView
 
 from course_discovery.apps.core.models import Partner
-from course_discovery.apps.edly_discovery_app.api.v1.tasks import run_dataloader
+from course_discovery.apps.edly_discovery_app.tasks import run_dataloader
```

Comment:
Just curious, why did we move it from api to outside? 🤔

Reply:
tasks.py should be on app's root, good practice.

Reply:
Okay yeah, that makes sense